src/WebJobs.Extensions.DurableTask/DurableOrchestrationContext.cs

```diff
@@ -97,9 +100,8 @@ public string GetRawInput()
         /// <returns>
         /// The parsed <c>JToken</c> representation of the orchestrator function input.
         /// </returns>
-        public JToken GetInputAsJson()
+        internal JToken GetInputAsJson()

```

Comment:
Were you following my GitHub thread with Brandon? :)

Reply:
Caught!